src/Aspire.Hosting/Dcp/ApplicationExecutor.cs

```diff
@@ -685,6 +685,7 @@ private void ConfigureAspireDashboardResource(IResource dashboardResource)
 
             context.EnvironmentVariables["ASPNETCORE_URLS"] = appHostApplicationUrl;
             context.EnvironmentVariables["DOTNET_RESOURCE_SERVICE_ENDPOINT_URL"] = grpcEndpointUrl;
+            context.EnvironmentVariables["DOTNET_RESOURCE_SERVICE_DISABLE_AUTH"] = "1"; // No auth in local dev experience
```

Comment:
What's the prior art on 1/0 vs true/false?

I prefer true/false strings but my mind is open.

Reply:
In the Aspire repo, `DOTNET_WATCH` uses `0` and `1`, so I followed that pattern for consistency. The `GetBool` extension method will handle `true` and `false` values too, however. I don't have a strong opinion either way.

Reply:
I would use true/false. 1/0 would be supported by nature of using that helper.

Reply:
`GetBool` supporting reading 0/1 is ok, but I think our docs should always show true/false, and values set by Aspire host should be true/false.

Reply:
I see more precedence for 0/1 than for true/false, so I'm inclined to leave this as is.

- `DOTNET_CLI_TELEMETRY_OPTOUT ` https://learn.microsoft.com/en-us/dotnet/core/tools/telemetry#how-to-opt-out
- `DOTNET_RUNNING_IN_CONTAINER` https://learn.microsoft.com/en-us/dotnet/core/tools/dotnet-environment-variables#jit-memory-barriers
- `DOTNET_EnableDiagnostics` https://learn.microsoft.com/en-us/dotnet/core/tools/dotnet-environment-variables#dotnet_enablediagnostics
- `DOTNET_MULTILEVEL_LOOKUP` https://learn.microsoft.com/en-us/dotnet/core/tools/dotnet-environment-variables#dotnet_multilevel_lookup

Reply:
It's a bit of a mess really. Looking through https://learn.microsoft.com/en-us/dotnet/core/tools/dotnet-environment-variables, you see some that take true/false, but most of them use 1/0.

Reply:
So do we want to break with the norm and use true/false in docs/templates/playgrounds/samples/etc?

Reply:
I'm going to convert these to values like `ResourceServiceClient:Insecure:DisableAuth` so will doc as `true`, seeing as the 0/1 seems more aligned with the env var presentation in .NET.

Reply:
I agree